bundles/org.openhab.core.audio/pom.xml

```diff
@@ -14,21 +14,34 @@
 
   <name>openHAB Core :: Bundles :: Audio</name>
 
+  <properties>
+    <bnd.importpackage>!javazoom.*</bnd.importpackage>
```

Comment:
These properties are not used in openhab-core ([openhab-addons POM only](https://github.com/openhab/openhab-addons/blob/43fc81f562ddb44de8cf8202cd06ff737947eb6f/pom.xml#L145)). In openhab-core bnd.bnd files are used for  for bnd bundle specific configuratoins. But it seems unnecessary since these imports are not added to the manifest?

Reply:
They were added until I added that line... Maybe I am wrong? Can check once more.